ros2/src/airsim_ros_pkgs/src/airsim_ros_wrapper.cpp

```diff
@@ -1352,6 +1365,39 @@ std::shared_ptr<sensor_msgs::msg::Image> AirsimROSWrapper::get_depth_img_msg_fro
     return depth_img_msg;
 }
 
+std::shared_ptr<sensor_msgs::msg::CompressedImage> AirsimROSWrapper::get_jpeg_msg_from_img_msg(
+    std::shared_ptr<sensor_msgs::msg::Image> image_message)
+{
+    auto jpeg_message =
+        std::make_shared<sensor_msgs::msg::CompressedImage>();
+    jpeg_message->header.frame_id = image_message->header.frame_id;
+    jpeg_message->header.stamp = image_message->header.stamp;
+
+    jpeg_message->format = image_message->encoding;
+    jpeg_message->format += "; jpeg compressed ";
+    jpeg_message->format += image_message->encoding;
+
+    try {
+        cv_bridge::CvImageConstPtr cv_ptr = cv_bridge::toCvShare(image_message, image_message->encoding);
+
+        std::vector<int> params{cv::IMWRITE_JPEG_QUALITY, 100};
+        if (cv::imencode(".jpg", cv_ptr->image, jpeg_message->data, params)) {
+            image_metadata::Image image(jpeg_message->data.data(), jpeg_message->data.size());
+            image_metadata::Metadata metadata{0.1, {1, 2, 3}, {1,2,3}, 0.1, {1,2,3}, 0, 0.1, 0.1, "mission", 14, 10, 20, 0.1, 10.1, 20, {1,2,3}, 40, {1,2,3}, 10, 20, 0.1, std::chrono::system_clock::now(), 0.1};

```

Comment:
This is just nonsense metadata for now since this PR is just to add the ability to publish jpegs with metadata.  Filling out metadata from Airsim telemetry will come in a following PR.